modules/nats/nats_test.go

```diff
@@ -57,3 +59,52 @@ func TestNATS(t *testing.T) {
 
 	require.Equal(t, "hello", string(msg.Data))
 }
+
+func TestNATSWithConfigFile(t *testing.T) {
+	const natsConf = `
+listen: 0.0.0.0:4222
+authorization {
+    token: "s3cr3t"
+}
+`
+	ctx := context.Background()
+
+	//  createNATSContainer {
+	ctr, err := tcnats.Run(ctx, "nats:2.9", tcnats.WithConfigFile(strings.NewReader(natsConf)))
+	//  }
```

Comment:
Ups! sorry but last minute change: we use these markers for embedding code in the docs site. Unfortunately this is already in use, so I'd advocate for 1) removing it or 2) renaming and adding a code snippet in the module docs.

Reply:
I though those are folding markers for $EDITOR :-)

Replaced by regular comments: https://github.com/testcontainers/testcontainers-go/pull/2905/commits/554f949ca217ceb59479e4dca08a1c7b17cde7f1